docs/src/_data/conversions.json

```
@@ -0,0 +1,45 @@
+{
+    "toNpmCommands": {
+        "install": "install",
+        "run": "run",
+        "init": "init",
```

```suggestion
        "install": "install",
        "init": "init",
```

we can remove `run` form here also.